sdk/tests/core/test_az_logging.c

```diff
@@ -66,6 +66,30 @@ static void _log_listener(az_log_classification classification, az_span message)
   }
 }
 
+static bool _should_write_everything_valid(az_log_classification classification)
```

Comment:
To match the `_should_write_any_mqtt()`, shouldn't this one be called `_should_write_any_http()`?

Reply:
I would like to leave this as is, especially since this is just naming in tests.